app/controller/button/DrawingButtonController.js

```diff
@@ -97,19 +103,126 @@ Ext.define('CpsiMapview.controller.button.DrawingButtonController', {
             return ol.events.condition.primaryAction(evt) && ol.events.condition.noModifierKeys(evt);
         };
 
+        me.prepareDrawingStyles();
+
+        var source = layer.getSource();
+        var collection = source.getFeaturesCollection();
         var drawInteractionConfig = {
             type: 'LineString',
-            features: layer.getSource().getFeaturesCollection(),
+            features: collection,
             condition: drawCondition,
+            style: me.getDrawStyleFunction(),
             snapTolerance: 2 // Pixel distance for snapping to the drawing finish (default 12)
         };
 
         me.drawInteraction = new ol.interaction.Draw(drawInteractionConfig);
+        me.drawInteraction.on('drawstart',me.handleDrawStart);
```

Comment:
Needs a space after comma to pass linting?

Reply:
fixed with [abc05db](https://github.com/compassinformatics/cpsi-mapview/pull/578/commits/abc05dba3e376f274ce2ecd1c8be494da91234b0)